eZ/Publish/SPI/Persistence/Content/Section/Handler.php

```diff
@@ -94,4 +94,22 @@ public function assign($sectionId, $contentId);
      * @return int
      */
     public function assignmentsCount($sectionId);
+
+    /**
+     * Number of role policies using a Section in limitations.
+     *
+     * @param mixed $sectionId
+     *
+     * @return int
+     */
+    public function policiesCount($sectionId);
+
+    /**
+     * Counts the number of role assignments using section with $sectionId in their limitations.
+     *
+     * @param int $sectionId
+     *
+     * @return int
+     */
+    public function roleAssignmentsUsingSectionCount($sectionId);
```

Comment:
UsingSectionCount? sounds like we're getting role Assigments using a "Section Count" as if count was some attr of the section like the id... don't you think? i'd go for ```countRoleAssignmentsUsingSection``` maybe...